index.html

```diff
@@ -46,132 +46,55 @@
 	</div>
 
-	<div id="scheduling-framework" class="block scheduling">
+	<div id="current-development" class="block scheduling">
 		<div style="background-image: none" class="container">
 
-			<h2><img src="assets/img/scheduling-framework.svg" /> Scheduling Framework</h2>
-
-			<div class="row">
-				<div class="col-sm-12">
-					<p class="block-description">The following components provide a scheduling framework to help you easily build custom tooling,
-						such as schedulers, on top of Amazon ECS. The framework makes it easy to consume events from Amazon ECS, store the cluster state locally, and
-						query the local data store though APIs. </p>
-				</div>
-			</div>
-
-			<div class="custom-panel">
-				<div class="row">
-					<div class="col-sm-3">
-						<h3><img src="assets/img/repo.svg" /> <a href="https://github.com/blox/blox/tree/master/cluster-state-service">cluster-state-service</a></h3>
-					</div>
-					<div class="col-sm-9">
-						<p>The <a href="https://github.com/blox/blox/tree/master/cluster-state-service">cluster-state-service</a> consumes events from a stream of all changes to containers and instances across your Amazon ECS clusters,
-							persists the events in a local data store, and provides APIs (e.g., search, filter, list, etc.) that enable you to query the state of your cluster
-							so you can respond to changes in real-time. The cluster-state-service tracks your Amazon ECS cluster state locally,
-							and manages any drift in state by periodically reconciling state with Amazon ECS. </p>
-					</div>
-				</div>
-				<div class="spacer-40"></div>
-				<div class="row">
-					<div class="col-sm-3">
-						<h3><img src="assets/img/repo.svg" /> <a href="https://github.com/blox/blox/tree/master/daemon-scheduler">daemon-scheduler</a></h3>
-					</div>
-					<div class="col-sm-9">
-						<p>The <a href="https://github.com/blox/blox/tree/master/daemon-scheduler">daemon-scheduler</a> is a scheduler that allows you to run exactly one task per host across all nodes in
-						a cluster. The scheduler monitors the cluster state and launches tasks as new nodes join the cluster, and
-						it is ideal for running monitoring agents, log collectors, etc. The scheduler can be used as a reference for
-						how to use the cluster-state-service to build custom scheduling logic, and we plan to add additional
-						scheduling capabilities for different use cases. </p>
-					</div>
-				</div>
-			</div>
-
-		</div>
-	</div>
-
-	<div id="command-line-tools" class="block cli">
-		<div class="container">
-
-			<h2><img src="assets/img/command-line-tools.svg" /> Command Line Tools</h2>
+			<h2><img src="assets/img/releases-and-roadmap.svg"/> Current Development</h2>
 
 			<div class="row">
-				<div class="col-md-7">
-					<p class="block-description">The following project provides local development tools for Amazon ECS.</p>
+				<div class="col-md-10">
+					<p class="block-description">Blox v1.0 is under active development.
```

Comment:
What is current available for use? Nothing? What version number are we on right now?